notebooks/COS/DayNight/DayNight.ipynb

```diff
@@ -0,0 +1,621 @@
+    "ax1.legend(fontsize = 20)\n",
+    "ax2.legend(fontsize = 20)\n",
+    "ax0.set_ylim(-2E-14, 6.5E-13)\n",
+    "ax1.set_ylim(-2E-14, 6.5E-13)\n",
+    "ax2.set_ylim(ax1.get_ylim())\n",
+    "\n",
+    "ax0.set_title(\"Fig 2.1\\nSpectra of Filtered and Unfiltered by `SUN_ALT`\", size = 35)\n",
+    "ax1.set_ylabel(\"Flux [$\\AA$]\", fontsize = 30, y=0.2, horizontalalignment='center')\n",
+    "ax2.set_xlabel(\"Wavelength [$\\AA$]\", fontsize = 30)\n",
+    "\n",
+    "plt.tight_layout()\n",
+    "\n",
+    "plt.savefig(plots_dir + 'compare_spectra_sunalt.png', dpi = 300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**We can see that the filtered spectrum largely follows the unfiltered spectrum; however, we significantly reduce the Lyman-alpha light from airglow around 1215 Å.**\n",
+    "\n",
+    "Because we filter out many of the datapoints used to calculate the spectrum, we can see a significant reduction in precision in flux space (visible as a **banding** in the y-axis). This can come about because with few datapoints, the bands represent wavelengths which received (0,1,2...n) discrete photons. The banding can sometimes be more pronounced at longer, redder wavelengths. \n",
+    "\n",
+    "**Below, let's make one final plot: a chunk of the spectrum around \\~1215 Å showing the errors in flux:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, (ax0) = plt.subplots(nrows=1, ncols=1, figsize=(15,8))\n",
+    "\n",
+    "ax0.errorbar(combo_dict_u[\"WAVELENGTH\"], combo_dict_u[\"FLUX\"], combo_dict_u[\"ERROR\"], \n",
+    "             linestyle = '', markersize = 2, alpha = 1, c = 'C1',label = \"Unfiltered\")\n",
+    "ax0.errorbar(combo_dict_f[\"WAVELENGTH\"], combo_dict_f[\"FLUX\"], combo_dict_f[\"ERROR\"], \n",
+    "             linestyle = '', markersize = 2, alpha = 1, c = 'C2', label = \"Filtered to SUN_ALT<0\")\n",
+    "\n",
+    "ax0.legend(fontsize = 20)\n",
+    "ax0.set_xlabel(\"Wavelength [$\\AA$]\", fontsize = 30)\n",
+    "ax0.set_ylabel(\"Flux [$\\AA$]\", fontsize = 30, horizontalalignment='center')\n",
+    "ax0.set_title(\"Fig 2.2\\nSpectra with Errors of Filtered and Unfiltered by `SUN_ALT`\", size = 30)\n",
+    "ax0.set_xlim(1213,1218)\n",
+    "\n",
+    "plt.tight_layout()\n",
+    "plt.savefig(plots_dir + 'ebar_compare_spectra_sunalt.png', dpi = 300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**With substantially fewer datapoints, our filtered dataset has larger errors.**\n",
+    "\n",
+    "We can, however, understand why we might want to filter by this, or another `sun_alt` filter. For instance, if most of your exposure was taken at night, but the last 10% was taken after the sun had risen and induced an atmospheric line which interferes with your data, it would often be necessary to filter out this last 10% of the exposure."
```

Comment:
You can ignore this if you wish since I'm not following the "science" story closely, but... This notebook sort of lacks a conclusion.  I.e., what did we learn from doing this day/night split?  Even though it's sorta in the plots it's probably worth having a short conclusion on the topic.